includes/Settings/SettingsGeneral.php

```diff
@@ -197,6 +197,19 @@ public function init_settings() {
 					'description'  => __( 'Required for UBL output format.<br>You can display this number on the invoice from the document settings.', 'woocommerce-pdf-invoices-packing-slips' ) . ' ' . $requires_pro        ,
 				)
 			),
+			array(
+				'type'     => 'setting',
+				'id'       => 'shop_phone_number',
```

Comment:
Maybe without the `shop_` prefix like the others?

Reply:
I actually wanted to make it similar to the `shop_address`, but without the prefix, it would also be good.

Reply:
I see. I agree with you then.

Reply:
Done.